yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/AttemptsConfiguration.java

```diff
@@ -0,0 +1,75 @@
+package com.yoti.api.client.docs.session.create;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class AttemptsConfiguration {
```

Comment:
don't know if u're constrained here but there are very long names in here

Reply:
I've reduced them down a little bit, but so that they still make sense to the integrator.